pkgs/desktops/plasma-5.1/startkde/startkde.in

```diff
@@ -0,0 +1,361 @@
+#!@bash@/bin/bash -x
+#
+#  NIXOS KDE STARTUP SCRIPT
```

Comment:
Why is this (very large) script needed? Doesn't upstream provide a startup script? It did in KDE 4 (`${kde_workspace}/bin/startkde`).